### PR TITLE
chore: PAYG banner

### PR DESCRIPTION
**Pull request by ShmuelLotman**
Screen Shot 2021-10-01 at 10 41 18 AM
Closes https://app.zenhub.com/workspaces/cloud-growth-5ff4b2ef8980820015495697/issues/influxdata/cloud-growth/155

Adds that little banner guy right up in the top right

### CODE REVIEW

File: src/checkout/CheckoutForm.scss [hunk 1]
```diff
@@ -14,6 +16,26 @@
   min-height: 226px;
 }
 
+.checkout-form--banner {
+  position: absolute !important;
+  right: -90px;
+  top: 19px;
+  rotate: 45deg;
+  width: 300px;
```
Comment (asalem1) [inline src/checkout/CheckoutForm.scss:24]: can we not do this with relative positioning?

File: src/checkout/CheckoutForm.tsx [hunk 2]
```diff
@@ -80,6 +83,13 @@ const CheckoutForm: FC = () => {
                 </a>
                 .
               </p>
+              <Panel
+                gradient={Gradients.HotelBreakfast}
+                className="checkout-form--banner"
+              >
+                <strong className="checkout-banner--credit">$250</strong>
```
Comment (asalem1) [inline src/checkout/CheckoutForm.tsx:90]: Is this number an actual thing?

**Review by asalem1 — COMMENTED** [review]
Do we have events in place to capture the click rate of the form so that we can compare the impact this has?

File: src/checkout/CheckoutForm.tsx (hunk 2 above)
Comment (ShmuelLotman) [reply src/checkout/CheckoutForm.tsx:90]: YYes

**Review by drdelambre — CHANGES_REQUESTED** [review]
The css here is too specific with the positioning layout and won't stay consistent as screen sizes change.

**Comment by ui-ci-bot-influx** [comment]
Metric | Score | Level
------------ | ------------- | ------------- 
performance | 62 | :yellow_circle:
accessibility | 96 | :green_circle:
seo | 90 | :green_circle:
pwa | null | :hankey:
best-practices | 100 | :green_circle:


[Compare your scores with master branch](https://influxdb.aws.influxdata.io/orgs/844910ece80be8bc/dashboards/073c4b95f1089000?lower=now%28%29%20-%202d&vars%5Blighthouse_score_branch%5D=2760)

**Review by asalem1 — APPROVED** [review]
"much gooder"

**Review by drdelambre — APPROVED** [review]
this is great. it decouples the parent from the child's designs so they can be altered independently

**Pull request merged**